Support for validation and multiple runs

6950167 opened the pull request.

OGKushper committed:
Server: Added support for multi runs in evaluation tranfer

OGKushper committed:
Merge branch 'dev' into evaluation-multirun

OGKushper committed:
more fixes and some test print statements to resultbyid

6950167 committed:
frontend support for multiple runs

6950167 committed:
Merge branch 'Results_Yme' of https://github.com/TheMinefreak23/fair-rec-kit-app into Results_Yme

6950167 committed:
CLIENT: bugfix + table sort indication

OGKushper committed:
Fixed incorrect dataset & recommender approach indices bug

6950167 committed:
CLIENT: changed sort icons, fixed bug

6950167 committed:
Merge remote-tracking branch 'origin/evaluation-multirun' into Results_Yme

6950167 committed:
CLIENT/SERVER validating fully functional

6950167 committed:
CLIENT: Start on unit tests (resultFormatter.test)

6950167 committed:
Merge branch 'dev' into Results_Yme

6950167 committed:
attempt at a merge conflict (not functional)

6950167 committed:
CLIENT: Actually fixed merge conflict

6950167 committed:
Merge branch 'dev' into Results_Yme

mendoxei commented:
I get some errors when doing a new experiment now, like this one
![image](https://user-images.githubusercontent.com/14370250/172019336-7053e30c-5312-4b12-a3fc-f24bdc4e2580.png)
Will probably investigate tomorrow

Still VERY juicy stuff tho

mendoxei committed:
CLIENT & SERVER: fixed multiple runs (evals)

Runs are stored in the metadata (in result_by_id) rather than assumed from the metrics, so experiments without metrics work too

mendoxei committed:
CLIENT: type match fix

mendoxei reviewed (approved):
fixed buddyyy
